src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  box-sizing: border-box;
+  background: $card-background-color;
+  display: flex;
+  flex-wrap: wrap;
+  width: 200px;
+  border: 1px solid #F3F3F3;
+  border-radius: 5px;
+  font-weight: 500;
+  transition: all $transition-duration;
```

Comment:
Specify to which properties the transition should apply instead of using `all`:
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x